nautobot/project-static/css/dark.css

```diff
@@ -165,4 +165,27 @@ html[data-theme="dark"] .filter-container .filter-selection {
 html[data-theme="dark"] .filter-container .filter-selection .filter-selection-choice {
   background-color: rgb(185, 185, 185);                 /* Background color of dynamic filters selected across most pages, inner selection box */
   border: 1px solid rgb(170, 170, 170);                 /* Border color of dynamic filters selected across most pages, inner selection box */
-}
\ No newline at end of file
+}
+
+
+/* Buttons */
+html[data-theme="dark"] .btn-default {
+    background-color: rgb(38, 38, 38) !important;
+}
+
+
+/* List View Table and Tables in Detail Views */
+html[data-theme="dark"] .table-responsive>table>tbody>tr,
```

Comment:
This looks like it may no longer be correct after #4775 and especially [the change to use `panel_table.html` instead of `responsive_table.html`](https://github.com/nautobot/nautobot/pull/4775/files#diff-dcf14e9be2ee02d07147dd5a769afd96900a4fdd3e51fdc73498b9e1583787b9L129-R129). May need to revise this.

Reply:
Any update here?

Reply:
Using `.table-responsive>table>tbody>tr` will affect multiple tables in Nautobot, but I believe `.table-headings>tbody>tr` will only affect the navbar menu drop down entries.